src/relentless/simulate/lammps.py

```diff
@@ -1313,7 +1363,20 @@ def _post_run_commands(self, sim, sim_op):
         return cmds
 
     def process(self, sim, sim_op):
-        pass
+        file_format = analyze.WriteTrajectory._detect_format(
+            sim.directory.file(self.filename), self.format
+        )
+        if file_format == "HOOMD-GSD":
+            if mpi.world.rank_is_root:
+                gsd_file = sim.directory.temporary_file(".gsd")
+                with gsd.hoomd.open(gsd_file, "wb") as t:
+                    for snap in lammpsio.DumpFile(
+                        sim.directory.file(self.filename), sort_ids=True
+                    ):
+                        frame = snap.to_hoomd_gsd()
```

Comment:
Do we need to override the types in the GSD frame, if there is a typemap set for LAMMPS? I think so and I again I forgot that.